ethereum/statetransition/src/main/java/tech/pegasys/artemis/statetransition/StateTransition.java

```diff
@@ -226,15 +224,14 @@ public void process_slots(MutableBeaconState state, UnsignedLong slot, boolean p
             .plus(UnsignedLong.ONE)
             .mod(UnsignedLong.valueOf(SLOTS_PER_EPOCH))
             .equals(UnsignedLong.ZERO)) {
-          STATUS_LOG.log(
-              Level.INFO, "******* Epoch Event *******", printEnabled, StatusLogger.Color.BLUE);
+          STATUS_LOG.epochEvent();
```

Comment:
This is likely to trigger too often - there are multiple places in the codebase that will call `process_slots` but they aren't necessarily moving the canonical view of the chain forward.  Previously `printEnabled` appears to have always been false so this never printed. I suspect `printEnabled` should have been true some times but got broken somewhere along the way.  We probably want to print epoch events from `BeaconChainController.processSlot` where the slot events are currently printing as that is the canonical view moving forward.

Reply:
Yes, I'd noticed that `printEnabled` was always `false` ..which did look suspect. This input saves me from working backwards to figure out where to remove the unnecessary calls from  🙂